data/wanderer/wanderers start.txt

```diff
@@ -179,67 +183,388 @@ mission "Wanderers: Hai Diplomat"
+		to display
+			has "Wanderers: Diplomacy Delayed: offered"
+	`	"Good to see you again Captain <last>, I have arranged for some refreshments so please, come and tell me all about these new aliens you have discovered!"`
+	`	Her exuberance is catching and after recent stressful events you see a new side to her as you relay your experience with the bird-like aliens. You realize that opportunities like this are why she became a diplomat in the first place and that this is, to some degree, a dream come true.`
+	`	"So in conclusion, it seems you need a translator," she says with a wry smile.`
+	choice
+		`	"Indeed, I'm hoping you have someone in mind."`
+		`	"It would be great to work with you again if you're willing."`
+	`	You open your mouth to respond but the effort is pointless.`
+	`	"I accept!" she cries, leaning forward on the desk with a gleeful look in her eyes.`
+	`	"A first contact is a dream opportunity, and there are so very few of them. So of course I will come with you."`
+	choice
+		`	"Let's go visit these aliens immediately!"`
+			goto "let's go"
+		`	"Can a distinguished ambassador such as yourself just leave like that?"`
+		`	"What if the humans want our own envoy to these aliens?"`
+			goto envoy
+	`	"Yes," she says. "If these aliens will provide a place for me, I mean to live with them permanently, an ambassador between their people and ours. I am old, and in need of a great novelty like this to make my life interesting once more." You note that the fur on her face, especially on the snout, has turned from brown to silvery gray - a sign of age in terrestrial creatures, and perhaps in the Hai as well.`
+	# Sayari is the head of the entire Hai diplomatic corps, so she doesn't need permission to choose herself as the diplomat.
+	# Going to diplomatic hot-spots is literally her job.
+	choice
+		`	"Let's go visit these aliens immediately!"`
+			goto "let's go"
+		`	"What if the humans want our own envoy to these aliens?"`
+	label envoy
+	`	She says, "If they prove to be friendly, I will help you to find a way to communicate with them directly, so you need not use me as an intermediary. But our first step must be to speak with them." You help her aboard your ship, and get ready to return to the alien homeworld.`
+		goto "let's go"
+
+	label "let's go"
+	action
+		event "Wanderers after Hai Reveal"
+		fail "First Contact: Wanderer"
+	`	"Excellent! I will await you in your ship."`
+		accept
+
+	label "Alternative Ambassador"
+	action
+		log "People" "Irayas Payaana" `A Hai ambassador who made first contact with the Syndicate and "other hostile human groups." Sayari, the head of the Hai diplomatic corps, chose her as the Hai ambassador to the aliens to the north of Hai space.`
+		log "People" "Sayari" `The head of the Hai diplomatic corps who hand-picked Irayas Payaana as the Hai representative to the aliens north of Hai space. Her diplomatic corps believes you have "a lot of potential... and a jump drive."`
```

Comment:
In this scenario, jump drive is being used as an outfit name and should be capitalized, correct?

Reply:
I don't know the rules for that.

Reply:
I think @tibetiroka think about that most?

Reply:
Jump drive is one of the few outfits that doesn't get capitalized in this context, so the current version is correct.

Reply:
Could you resolve it?